src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/overlays/ExternalLinkTypeOverlay.js

```diff
@@ -169,11 +203,21 @@ class ExternalLinkTypeOverlay extends React.Component<LinkTypeOverlayProps> {
                         </Fragment>
                     }
 
-                    {onTitleChange &&
-                        <Form.Field label={translate('sulu_admin.link_title')}>
+                    {onTitleChange
+                        && <Form.Field label={translate('sulu_admin.link_title')}>
                             <Input onChange={onTitleChange} value={title} />
                         </Form.Field>
                     }
+
+                    {onRelChange
+                        && <Form.Field label={translate('sulu_admin.link_rel')}>
+                            <MultiSelect onChange={this.handleRelChange} values={this.relValue}>
+                                <MultiSelect.Option value="nofollow">nofollow</MultiSelect.Option>
+                                <MultiSelect.Option value="noreferrer">noreferrer</MultiSelect.Option>
+                                <MultiSelect.Option value="noopener">noopener</MultiSelect.Option>
```

Comment:
I implemented this as a `MultiSelect` for now, to be most flexible in the future, even though @alexander-schranz mentioned [here](https://github.com/sulu/sulu/issues/3668#issuecomment-1143579307), that he only wants to support `nofollow`. I'm open to change this to a checkbox to only allow `nofollow`, but the underlying logic should stay the same, that `rel` is passed around as `string` instead of a bool flag specificially for `nofollow`.

/cc @sulu/core-team Would you only allow `nofollow` or are there more options that would make sense? A list can be found here: https://www.w3schools.com/tags/att_a_rel.asp

Reply:
@luca-rath not sure but the only one which could make sense is `noreferrer` /cc @alexander-schranz